fission/vite.config.ts

```diff
@@ -37,34 +37,45 @@ if (useSsl) {
     plugins.push(basicSsl())
 }
 
+const localAssetsExist = await fs.access("./public/Downloadables/Mira",fs.constants.R_OK).then(() => true).catch(() => false)
+
+
 // https://vitejs.dev/config/
 export default defineConfig(({ mode, }) => {
```

Comment:
```suggestion
export default defineConfig(({ mode }) => {
```
I think that this comma isn't supposed to be there